src/frontends/jax/src/utils.cpp

```diff
@@ -16,6 +16,7 @@ namespace jax {
 void num_inputs_check(const NodeContext& context, size_t min_inputs, size_t max_inputs) {
     auto inputs = context.inputs();
     FRONT_END_OP_CONVERSION_CHECK(inputs.size() >= min_inputs, "Got less inputs than expected");
+    FRONT_END_OP_CONVERSION_CHECK(inputs.size() <= max_inputs, "Got more inputs than expected");
```

Comment:
Not sure if I can fix this here, or maybe it's better to move it to a separate PR?

Reply:
yes, let's do it.

Reply:
Thanks @rkazants Just to clarify, do you mean I should keep it here or I should move it to a separate PR?

Reply:
@rkazants it would be great if you  could please clarify this, thanks :)

Reply:
Yes, please add this check in this PR.